submit/templates/submit/jref.html

```diff
@@ -0,0 +1,136 @@
+{%- extends "base/base.html" %}
+
+{% import "submit/submit_macros.html" as submit_macros %}
+
+{% block addl_head %}
+ <link rel="stylesheet" href="{{ url_for('static', filename='css/submit.css')}}" />
+ <script src="{{ url_for('static', filename='js/filewidget.js') }}"></script>
+{% endblock addl_head %}
+
+{% block content %}
+  <h1 class="title">Update journal reference</h1>
+  <p>
+    When an article is published, the author may wish to indicate this in the
+    abstract listing for the article. For this reason, the journal reference
+    and DOI (Digital Object Identifier) fields are provided for articles.
+    Additionally, a report number field is provided for institutional report
+    numbers.
+  </p>
+  {% if require_confirmation and not confirmed %}
+  <div class="notification is-info">
+    Please review the preview of your paper's abstract page, and confirm that
+    you with to proceed. You can continue to make changes until you are
+    satisfied with the result.
+  </div>
+  {% endif %}
+  <h2>arXiv:{{ submission.arxiv_id }}v{{ submission.version }} {{ submission.metadata.title }}</h2>
+  <content>
+    <form method="POST" action="{{ url_for('ui.jref', submission_id=submission_id) }}">
+      <div class="columns">
```

Comment:
I really love the two-column layout in mockup, but this may not look accurate as a preview once we get the drop-in from browse. May have to consider vertically stacked boxes instead if the columns are too narrow to show an accurate preview. Maybe we can put the two instruction/description boxes in the right column and split it 2/3 : 1/3? Or something. We'll have to play with it a bit.